### ISSUE TITLE
Use scss instead of plain css

### ISSUE THREAD

**Issue by Paradyx** [RPGLibrarium/web#10]
Use scss magic to clean up code and make it more maintainable. Suggested by @Tikrass

**Comment by sudo42** [comment]
Hugo is able to process SCSS, but it seems to be meant for themes.
This would match my plan to move the "theme-componets" (which we are just overriding/shadowing at the moment) to a separate theme (+ repo?).

**Comment by Paradyx** [comment]
The theme will only be used with hugo. No need for an extra repository for now.

**Comment by sudo42** [comment]
True. But at least we could create a "local fork" of the current base theme inside of this repository instead of the current (rather unsatisfying) solution.
